app/controllers/companies_controller.rb

```diff
@@ -1,5 +1,6 @@
 class CompaniesController < ApplicationController
   before_action :set_company, only: %i[show edit update]
+  include ActiveStorageDownloader
```

Comment:
curious whey this is needed? I kinda forget what it does

Reply:
removing that, it was something I tried initially, but no longer needed